### ISSUE TITLE
Is velocyto available for single-nuclei data?

### ISSUE THREAD

**Issue by masato-tsutsui** [velocyto-team/velocyto.py#274]
I am trying to analyse RNA-velocity for single-nuclei RNA sequencing data from 10X platform. I am not sure in detail how it calculates the unspliced/spliced rate when I run the command 'velocyto run10X', especially about whether it considers the percentage of reads containing unspliced intronic sequences. And it was reported that the percentage varies among platforms in the paper '_RNA velocity of single cells_'.  Then I am wondering if that percentage differs between cell soma and nuclei beyond platforms. Can you kindly tell me some information about that?

**Comment by lianov** [comment]
Agreed that this would be a nice feature to have although understandably a potentially difficult one to support for nuclei data. Any updates on whether there will be support of RNA velocity analysis for single-nuclei?

**Comment by masato-tsutsui** [comment]
Hi, thank you for replying. Firstly I checked the unspliced proportion and found it was about 66%, which I assume is higher than that seen in single 'cell' cases. Although I have not checked into Issues in scvelo platform, some also reported that single 'nuclei' data had ~70% unspliced portions. 
     And when I applied the dynamical-mode velocity analysis available in the scvelo package, it successfully worked and found there were reasonable directions like immature cells-> mature cells or healthy cells -> disease cells. Other reseachers posted issues about the oppsite directions to their knowledge when used the dynamical-mode velocity analysis in the scvelo platform, but I did not see the erroneous results like that.
    So I am understanding single 'nuclei' data can be applied to velocity analysis, but I am still wondering whether the mathematical model including RNA degradation parameters can be straightly applied to 'nuclei' data because the degradation event should occur in the cell soma and this should not be captured in case of 'nuclei' data....
